src/dmd/vsoptions.d

```diff
@@ -569,12 +616,80 @@ private:
     }
 }
 
+private:
+
+inout(wchar)[] toDString(inout(wchar)* s)
+{
+    return s ? s[0 .. wcslen(s)] : null;
+}
+
+extern(C) wchar* _wgetenv(const(wchar)* name);
+
+char* getenv(wstring name)
+{
+    if (auto wide = _wgetenv(name.ptr))
+        return toNarrowStringz(wide.toDString).ptr;
+    return null;
+}
+
+char* buildPath(const(char)*[] fragments...)
```

Comment:
This being here has more to do with how this PR has evolved in ldc-developers/ldc#3415. Maybe it should replace the `FileName.buildPath()` implementation, which inefficiently uses N-1 buffers when specifying N fragments.